cartogram_generator/fill_with_density.c

```diff
@@ -298,7 +300,10 @@ BOOLEAN fill_with_density1 (char *map_file_name, char *area_file_name,
   while (fgets(line, MAX_STRING_LENGTH, area_file) != NULL) {
     id = -1;
     area = -1.0;
-    sscanf(line, "%d%*[, ]%lf", &id, &area);
+    r = 0.96;  
+    g = 0.92;
+    b = 0.70;
```

Comment:
After dividing by 255 in lines 315-317, the fill colors are essentially black, so the default fill color (black) is indistinguishable from the default border color (also black). Please change to r = 245.0, g = 235.0, b = 179.0.